package.json

```diff
@@ -8,7 +8,9 @@
   "type": "module",
   "exports": {
     ".": {
-      "import": "./lib/index.js"
+      "import": "./lib/herumi/index.js",
```

Comment:
shouldn't this be kept unchanged?

Reply:
The idea was to replicate the selection logic with conditional exports. So, unless it's node, it should fall back to herumi.
Ultimately I think we could completely drop off https://github.com/ChainSafe/bls/blob/master/src/index.ts

Reply:
But isn't this breaking change then?

Reply:
So one change I noticed now is that `node` targets w/o native available would not fallback to herumi with this scheme.
Looks like the reasonable option is to define a `browser` condition that bypass the check, and let the rest untouched.

Or did you have another thing in mind?

Reply:
> Or did you have another thing in mind?

Could still have the node specific export, this would make it more explicit if we know we only need to run a specific code in nodejs

Reply:
Not available probably, I don't think you can finish yarn install if bls fails to compile, but since this is a optional peer dep makes sense, it might not be installed

https://github.com/ChainSafe/bls/blob/f30cd5c9677b4d23a9d426e22375254e93ebd076/package.json#L122-L127